src/pages/homePage.js

```diff
@@ -1,6 +1,12 @@
 class HomePage {
```

Comment:
Please rename all files under `pages/` node. They should start from the capital letter.
Now you have `src/pages/accessoriesPage.js`
Should be `src/pages/AccessoriesPage.js`